integr8ly/application-monitoring-operator
# Os4 monitoring resources

Updates for OS4:

* use latest grafana operator and update config
* remove TLS from blackbox target config
* fix: CRs lost owner reference when updated, this prevented clean uninstall

Verification steps:

1. Create a project named `application-monitoring`
2. Create all [roles](https://github.com/integr8ly/application-monitoring-operator/tree/master/deploy/roles)
3. If not running locally, create all [operator roles](https://github.com/integr8ly/application-monitoring-operator/tree/master/deploy/operator_roles)
4. Deploy the monitoring stack by creating an application monitoring cr
5. Make sure that Prometheus, Grafana and Alertmanager come up successfully
6. Make sure that you can login to each of them using the routes
7. Login to Grafana using the admin credentials (found in a secret named admin-credentials)
8. Check that the Prometheus data source has been imported and that it can be validated successfully
